ui/lib/kv/addon/templates/secret/details/edit.hbs

```diff
@@ -0,0 +1,6 @@
+<Page::Secret::Edit
+  @newVersion={{this.model.newVersion}}
+  @previousVersion={{this.model.secret.version}}
+  @metadata={{this.model.metadata}}
```

Comment:
note to self, probably can just pass `metadata.currentVersion` but waiting until I've wrangled permissions